chcemvediet/apps/obligees/tests/test_models.py

```diff
@@ -1,369 +1,672 @@
+        oblg = self._create_obligee(omit=[u'simple_description'])
+        self.assertEqual(oblg.simple_description, u'')
 
-    @unittest.skip(u'FIXME')
     def test_status_field(self):
         tests = (
-                (Obligee.STATUSES.PENDING, u'Pending'),
-                (Obligee.STATUSES.DISSOLVED, u'Dissolved'),
+                (u'Test Name 1', Obligee.STATUSES.PENDING, _(u'obligees:Obligee:status:PENDING')),
+                (u'Test Name 2', Obligee.STATUSES.DISSOLVED, _(u'obligees:Obligee:status:DISSOLVED')),
                 )
         # Make sure we are testing all defined obligee statuses
-        tested_statuses = [a for a, _ in tests]
+        tested_statuses = [a for n, a, i in tests]
         defined_statuses = Obligee.STATUSES._inverse.keys()
         self.assertItemsEqual(tested_statuses, defined_statuses)
 
-        for status, expected_display in tests:
-            oblg = self._create_obligee(status=status)
+        for name, status, expected_display in tests:
+            oblg = self._create_obligee(name=name, status=status)
             self.assertEqual(oblg.status, status)
             self.assertEqual(oblg.get_status_display(), expected_display)
 
-    @unittest.skip(u'FIXME')
     def test_status_field_may_not_be_omitted(self):
-        with self.assertRaisesMessage(IntegrityError, u'obligees_obligee.status may not be NULL'):
+        with self.assertRaisesMessage(IntegrityError, u'NOT NULL constraint failed: obligees_obligee.status'):
             self._create_obligee(omit=[u'status'])
 
+    def test_notes_field(self):
+        oblg = self._create_obligee(notes=u'Notes about agency.')
+        self.assertEqual(oblg.notes, u'Notes about agency.')
+
+    def test_notes_field_default_value_if_omitted(self):
+        oblg = self._create_obligee(omit=[u'notes'])
+        self.assertEqual(oblg.notes, u'')
+
+    def test_historical_obligee_model_exists(self):
+        oblg = self._create_obligee(name=u'Agency', street=u'Westside')
+        count = HistoricalObligee.objects.filter(id=oblg.pk).count()
+        self.assertEqual(count, 1)
+
+    def test_history_records_changes_to_official_name_field(self):
+        oblg = self._create_obligee(official_name=u'Agency Official')
+        oblg.official_name = u'Secret Agency Official'
+        oblg.save()
+        history = oblg.history.all()
+        self.assertEqual([h.official_name for h in history], [u'Secret Agency Official', u'Agency Official'])
+
+    def test_history_records_changes_to_name_field(self):
+        oblg = self._create_obligee(name=u'Agency')
+        oblg.name = u'Secret Agency'
+        oblg.save()
+        history = oblg.history.all()
+        self.assertEqual([h.name for h in history], [u'Secret Agency', u'Agency'])
+
+    def test_history_records_changes_to_slug_field_when_name_field_changes(self):
+        oblg = self._create_obligee(name=u'Agency')
+        oblg.name = u'Another Agency'
```

Comment:
@viliambalaz Nevolaj ju v testoch "Another Agency", lebo to nie je ina institucia, iba iny nazov (hodnota fieldu) pre tu istu instituciu. Pouzi analogiu z predosluch testov "Agency" zmenene na "Secret Agency".